projects/demo/src/modules/components/input-time/input-time.template.html

```diff
@@ -155,6 +156,14 @@ <h3>DI-tokens for input-configurations:</h3>
             >
                 Items to choose
             </ng-template>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="itemsHidden"
+                documentationPropertyType="boolean"
+                [(documentationPropertyValue)]="itemsHidden"
+            >
+                Disable the display of the dropdown list
```

Comment:
Should we just use `inject(TuiDropdownOpen, {optional: true}).tuiDropdownEnabled` instead of creating new prop ? 🤔

Reply:
I think most people are on version 3, so it would be great to have this option already.